build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.android.tools.build:gradle:4.0.0'
```

Comment:
Please do not add unrelated changes like this. Gradle updates can sometimes cause issues and I only want to update when there is a specific bug that you experience with the old version.